### DISCUSSION TITLE
Bladbite Cuda plots slow vs. gigahorse/madmax

### DISCUSSION THREAD

**Discussion by scoville07** [Chia-Network/bladebit#378] [Q&A]
Hello,

I would like to replot with official bladebit. My plotter (Xeon E5-2697v4, 512GB RAM, RTX3060Ti, 2TB NVMe) was able to create gigahorse/madmax plots in about 2,5 minutes. Bladebit takes around 6 minutes. Is that normal or is there a way to tune that?

Thanks
Christian

**Comment by hajes** [comment]
I used to have around 4.5 minutes on same HW in Debian. <3min now in Clear Linux.
